examples/httpserver/bin/basic_writer_client.dart

```diff
@@ -20,11 +21,11 @@ Future main() async {
   };
 
   var request = await new HttpClient()
-      .post(InternetAddress.LOOPBACK_IP_V4.host, 4049, '/file.txt');
-  request.headers.contentType = ContentType.JSON;
-  request.write(JSON.encode(jsonData));
-  HttpClientResponse response = await request.close();
-  await for (var contents in response.transform(UTF8.decoder)) {
+      .post(InternetAddress.LOOPBACK_IP_V4.host, 4049, 'file.txt'); /*1*/
```

Comment:
At the window width I use when reviewing (just barely 3-column), this line is cut off. The last fully visible character is the `;`.

Reply:
fixed